.gitignore

```diff
@@ -1,12 +1,4 @@
-*.sublime-project
-*.sublime-workspace
-.idea
+/vendor/
 /.php_cs.cache
-/nbproject/private/
-/build/
-/cache/
-vendor
-report
-composer.lock
-phpunit.xml
-humbuglog.txt
+/composer.lock
+/phpunit.xml
```

Comment:
Maybe .dist and rename the file?

Reply:
I don't get the question - we have `.dist` file and ignore `phpunit.xml`, what would you like to change?

Reply:
Name it phpunit.xml.dist as it’s a best practice AFAIK and the ignore that file

Reply:
I still don't get it, what do you want to name phpunit.xml.dist?

AFAIK best practice is to have `phpunit.xml.dist` and ignore `phpunit.xml` - exactly what we have right now.

Reply:
You are totallly right I mixed things up 😂 Christmas is coming... need to relax more 😉